.eslintrc.js

```diff
@@ -11,36 +11,42 @@ module.exports = {
     browser: true,
     node: true,
   },
-  extends: ['plugin:import/recommended', 'airbnb', 'prettier', 'prettier/react'],
-  parser: 'babel-eslint',
+  extends: [
+    'plugin:import/recommended',
+    'plugin:import/typescript',
+    'airbnb-typescript',
```

Comment:
What the motivation of usage predefined config? Why not to use recommended by `@tpescript-eslint?`

Reply:
The end goal is to avoid developers or have to format the code once they copy and paste it from the demos. So the question we should answer to is: what's more popular in the community?

Reply:
Note that we were already using the Airbnb configuration. My objective was to cover TypeScript keeping the same rules, if we want to change the rule, that would be best in a different pull request.

Reply:
We can discuss the preset in another issue. This PR should be limited to switching the parser and adding TS files.